clients/java/src/main/java/io/camunda/client/api/search/response/Job.java

```diff
@@ -64,4 +64,8 @@ public interface Job {
   Long getElementInstanceKey();
 
   String getTenantId();
+
+  OffsetDateTime getCreatedAt();
+
+  OffsetDateTime getUpdatedAt();
```

Comment:
💭 To be consistent with the exsiting `endTime` attribute, we could also consider naming them `getCreationTime()` and `getLastUpdateTime()`.